### ISSUE TITLE
Look of the menu?

### ISSUE THREAD

**Issue by orschiro** [careteditor/caret#325]
Dear all, 

Is this how the menu is supposed to look like or is something wrong with my configuration on Ubuntu MATE 17.04?

![image](https://cloud.githubusercontent.com/assets/170947/25441790/829a996a-2aa3-11e7-9203-10cda4d806f8.png)

**Comment by astoilkov** [comment]
@orschiro - you are referring to the menu not being aligned with `Format` but instead with `Go` right? We don't have Ubuntu MATE machine to test and compare with other apps.

**Comment by orschiro** [comment]
@astoilkov, yes, this and the fact that it looks quite different from my other app menus:

**Example LibreOffice**

![image](https://cloud.githubusercontent.com/assets/170947/25523494/250339a0-2c07-11e7-882d-7446f0a712f1.png)

**Comment by pslobo** [comment]
Quick question. Given this is based on Electron, could you test to see if this also happens in Atom.io I think this could help weed out if it's an Electron issue or Caret issue.

**Comment by astoilkov** [comment]
@pslobo - great idea
@orschiro - can you test [Atom](https://atom.io/) or [VSCode](https://code.visualstudio.com/) to see if the same issue is present there. Thanks in advance.

**Comment by orschiro** [comment]
It's an Electron problem. 

Can we improve the formatting entry so that it opens to the bottom and not to the side?

Screencast: https://youtu.be/i9nkq_EJfVQ

**Comment by astoilkov** [comment]
As I can see the `Format` menu opens to the side because there is no available space. In Electron there is no option for this behavior.

Regarding the appearance issue - we will create an issue in Electron and post the updates here.

**Comment by orschiro** [comment]
>As I can see the Format menu opens to the side because there is no available space. In Electron there is no option for this behavior.

So, I just need to make the editor window bigger, correct?

> Regarding the appearance issue - we will create an issue in Electron and post the updates here.

Thank you!

**Comment by astoilkov** [comment]
> So, I just need to make the editor window bigger, correct?

Yes. I think this should fix it. Let me know.

**Comment by orschiro** [comment]
Yes, fixed. :-)
